Library/Formula/v8.rb

```diff
@@ -20,6 +22,8 @@ class V8 < Formula
   depends_on :macos => :lion
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
+  depends_on "pkg-config" => :build
+  depends_on "icu4c"
```

Comment:
Form what I read on chromium-packagers, most (all?) link `v8` statically, just so they have some control over API changes. Chromium on linux definitely links to system `icu` - that's where I'm nicking the gyp from :smile: 

I'm done for the day, but can look at making it optional tomorrow if you want.